examples/DDPG/train.py

```diff
@@ -0,0 +1,100 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import gym
+import numpy as np
+import time
+from mujoco_agent import MujocoAgent
+from mujoco_model import MujocoModel
+from parl.algorithms import DDPG
+from parl.utils import logger
+from replay_memory import ReplayMemory
+from ou_strategy import OUStrategy
+
+MAX_EPISODES = 5000
+MAX_STEPS_EACH_EPISODE = 1000
+ACTOR_LR = 1e-4
+CRITIC_LR = 1e-3
+GAMMA = 0.99
+TAU = 0.001
+MEMORY_SIZE = int(1e6)
+MIN_LEARN_SIZE = 1e4
+BATCH_SIZE = 128
+REWARD_SCALE = 1.0
+ENV_SEED = 1
+
+
+def main():
+    env = gym.make(args.env)
+    env = env.unwrapped
+    env.seed(ENV_SEED)
+
+    obs_dim = env.observation_space.shape[0]
+    act_dim = env.action_space.shape[0]
+    act_bound = env.action_space.high[0]
+
+    model = MujocoModel(act_dim, act_bound)
+    algorithm = DDPG(
+        model,
+        hyperparas={
+            'gamma': GAMMA,
+            'tau': TAU,
+            'actor_lr': ACTOR_LR,
+            'critic_lr': CRITIC_LR
+        })
+    agent = MujocoAgent(algorithm, obs_dim, act_dim)
+
+    rpm = ReplayMemory(MEMORY_SIZE, obs_dim, act_dim)
+
+    noise_strategy = OUStrategy(act_dim)
+
+    for i in range(MAX_EPISODES):
+        obs = env.reset()
+        noise_strategy.reset()
+        ep_reward = 0
+        for j in range(MAX_STEPS_EACH_EPISODE):
```

Comment:
`MAX_STEPS_EACH_EPISODE` is used to limit the maximum episode length here, I think it might be the reasons why this could't reach the performance of paper.
Are you sure that paper's reward is the total reward of an episode with 1000 steps at most?

Reply:
Yes, max episode length is 1000 in the papers.